docs/en/setup/service-agent/java-agent/Bootstrap-plugins.md

```diff
@@ -5,4 +5,17 @@ For using these plugins, you need to put the target plugin jar file into `/plugi
 Now, we have the following known bootstrap plugins.
 * Plugin of JDK HttpURLConnection. Agent is compatible with JDK 1.8+
 * Plugin of JDK Callable and Runnable. Agent is compatible with JDK 1.8+
-* Plugin of JDK ThreadPoolExecutor. Agent is compatible with JDK 1.8+
\ No newline at end of file
+* Plugin of JDK ThreadPoolExecutor. Agent is compatible with JDK 1.8+
+
+### Tips
+Plugin of JDK HttpURLConnection depended on `sun.net.*` ,If you use java9+, You may add some jvm options like:
+
+| java version | jvm option                                                               |
+|--------------|--------------------------------------------------------------------------|
+| 9-15         |nothing todo(because `--illegal-access` default model is permit)           |
+| 16           |`--add-exports java.base/sun.net.www=ALL-UNNAMED` or `--illegal-access=permit` |
+| 17+          |`--add-exports java.base/sun.net.www=ALL-UNNAMED`                              |
```

Comment:
```suggestion
| Java version | JVM option                                                               |
|--------------|--------------------------------------------------------------------------|
| 9-15         |Nothing to do. Because `--illegal-access` default model is permitted.           |
| 16           | Add`--add-exports java.base/sun.net.www=ALL-UNNAMED` or `--illegal-access=permit` |
| 17+          | Add `--add-exports java.base/sun.net.www=ALL-UNNAMED`                              |
```